### PR TITLE
DCOS-12334: split reducers

### PR DESCRIPTION
**Pull request by Poltergeist**
This introduces two separate reducers for apps and pods.

### CODE REVIEW

File: plugins/services/src/js/reducers/JSONPodReducers.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import {JSONReducer as constraints} from './serviceForm/Constraints';
```
Comment (orlandohohmeier) [inline plugins/services/src/js/reducers/JSONPodReducers.js]: Let's rename this file to `JSONMultiContainerReducers` to be consistent with the Volumes and Network reducers

**Comment by Poltergeist** [comment]
**mesosphere-ci** retest this please

**Comment by orlandohohmeier** [comment]
L👀ks good!

**Comment by wavesoft** [comment]
LGTM ✨

**Pull request merged**